package.json

```diff
@@ -51,6 +51,7 @@
     "prop-types": "15.8.1",
     "react": "16.14.0",
     "react-dom": "16.14.0",
+    "react-helmet": "^6.1.0",
```

Comment:
According to this blog https://www.freecodecamp.org/news/react-helmet-examples/ react-helmet is considered deprecated, and react-helmet-async https://github.com/staylor/react-helmet-async is the new version. Do you have any more information about this?

Reply:
Thanks, @cdeery for your review 

The main reason to decided to use react-helmet was other apps like `frontend-app-learning` were using it. https://github.com/openedx/frontend-app-learning/blame/master/package.json#L54 

I am not able to find more information about an officially deprecated announcement, but that seems to be deprecated  according to https://github.com/nfl/react-helmet/issues/689 (in addition the last commit date was 2020) and at the same time is still used for the community check date of this issue https://github.com/nfl/react-helmet/issues/693

We could make the change of the library to `react-helmet-async`

And @arbrandes this could be something to consider for the FWG, due to different apps are using this library.

Reply:
Yes, this looks like a concern for more repos than just this one.

It doesn't excuse us from doing the right thing, of course.  @cdeery, would an issue to tackle this across the board be enough to otherwise merge this as is?  This is something @abdullahwaheed and FED-BOM could probably help us with.

Reply:
I check how `react-helmet-async` could be used, for this specific case I think a new context is too much for the purpose of this change then I use another implementation [explained in the official repo](https://github.com/staylor/react-helmet-async#usage-without-context).

I create a commit to put this into consideration and find the best way to make the change to the library. :)

Reply:
Thanks @dcoa  for making this change. I think this is much better. I understand that other teams are using the older version, but this helps with not creating tech debt.
This generally looks ok to me. I am not sure what you mean by using a context is "too much", I would be surprised if it added a noticeable burden to the app. My inclination would be to use the library as indicated as the common pattern, because it looks like it would avoid any bugs that might occur if the explicit argument passing is missed.

Reply:
@cdeery I made the changes to use `HelmetProvider`, let me know what you think. :)